Code/Framework/AzCore/AzCore/Memory/AllocationRecords.cpp

```diff
@@ -16,8 +16,11 @@
 
 #include <AzCore/Debug/StackTracer.h>
 
-using namespace AZ;
-using namespace AZ::Debug;
+namespace AZ 
+{
+
+namespace Debug
+{
```

Comment:
Nit: the formatting here doesn't match the expected one ( namespace contents are indented )
If possible, I'd suggest using c++17 nested namespace feature : 
```
namespace AZ::Debug 
{

}
```